monitoring/types.go

```diff
@@ -0,0 +1,22 @@
+package monitoring
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	longhornName = "longhorn"
+
+	subsystemVolume      = "volume"
+	subsystemBackupStats = "backup_stats"
```

Comment:
remove the backup related field for now, since we are not doing it till later.

Reply:
Thank you! Fixed